src/test/java/com/linkedin/kmf/services/ConsumeServiceTest.java

```diff
@@ -106,6 +106,37 @@ public void commitAvailabilityTest() throws Exception {
     consumeService.awaitShutdown();
   }
 
+  @Test
+  public void commitLatencyTest() throws Exception {
+    CommitLatencyMetrics commitLatencyMetrics = Mockito.mock(CommitLatencyMetrics.class);
+    Assert.assertNotNull(commitLatencyMetrics);
+
+    ConsumeService consumeService = consumeService();
+
+    Metrics metrics = consumeService.metrics();
+    Map<String, String> tags = new HashMap<>();
+    tags.put(TAGS_NAME, TAG_NAME_VALUE);
+
+    Assert.assertNull(metrics.metrics().get(metrics.metricName("commit-offset-latency-ms-avg", METRIC_GROUP_NAME, tags)));
+    Assert.assertNull(metrics.metrics().get(metrics.metricName("commit-offset-latency-ms-max", METRIC_GROUP_NAME, tags)));
+
+    /* Should start */
+    consumeService.start();
+    Assert.assertTrue(consumeService.isRunning());
+
+    /* in milliseconds */
+    long threadStartDelay = 1000 * THREAD_START_DELAY;
```

Comment:
1. Can you rename the variable from `THREAD_START_DELAY ` to `THREAD_START_DELAY_SECONDS`?

2. Can you rename the variable from `threadStartDelay` to `threadStartDelayMs`? I think it is in general a good idea to let the variable name reflect the time unit.

3. Can you use `TimeUnit.SECONDS.toMillis(THREAD_START_DELAY_SECONDS);` to do the conversion?